package.json

```diff
@@ -28,7 +28,7 @@
     "clean": "rimraf target && rimraf dist && rimraf build2",
     "generate-phone-codes": "node buildtools generate-phone-codes",
     "generate-config": "node buildtools generate-language-config",
-    "retirejs": "retire --jspath src --package",
+    "retirejs": "retire --jspath target --package",
```

Comment:
could it be more specific to `target/js` ?

Reply:
Yes, that could save a few milliseconds